File: api/src/main/java/org/pmiops/workbench/exceptions/GatewayTimeoutException.java
package org.pmiops.workbench.exceptions;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(HttpStatus.GATEWAY_TIMEOUT)
public class GatewayTimeoutException extends WorkbenchException {
  public GatewayTimeoutException() {
    super();
  }
  public GatewayTimeoutException(String message) {
    super(message);
  }
}

File: api/src/main/java/org/pmiops/workbench/exceptions/GatewayTimeoutException.kt
package org.pmiops.workbench.exceptions

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(HttpStatus.GATEWAY_TIMEOUT)
class GatewayTimeoutException : WorkbenchException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}
}
